.github/ISSUE_TEMPLATE/COMMUNITY_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!-- (REQUIRED) What is the nature of this issue? -->
+## This Community doc request is a:
+- [ ] New topic request
+- [ ] Topic content updates
+- [ ] Other
+
+<!-- (REQUIRED) What is the Community issue and code PR? -->
+## Community GitHub issue link:
+
+## Community GitHub code PR link:
+
+<!-- (REQUIRED) What is the Magento Open Source/Commerce and version for the PR? Will it be back or forward ported? -->
+## Magento version:
+- [ ] Open Source
+- [ ] Commerce
+
+## Back and Forward porting:
```

Comment:
Is this really necessary? Isn't asking which version this applies to sufficient?

Reply:
Hmm, sometimes the code work is for a specific version, then may become back or forward ported. Will tweak/remove

Reply:
I asked for this because some code changes are version-specific. Also, the backports to big features can be a big surprise (such as bulk/async API), and it'd be really nice if we have forewarning about this.